docs/en/latest/concepts/apisix_route.md

```diff
@@ -184,29 +184,37 @@ If the Plugin is not enabled in APISIX by default, you can enable it by adding i
 
 :::
 
-The example below configures [cors](https://apisix.apache.org/docs/apisix/plugins/cors/) Plugin for the route:
+The example below configures [limit-count](https://apisix.apache.org/docs/apisix/plugins/limit-count) Plugin for the route:
 
 ```yaml
 apiVersion: apisix.apache.org/v2
 kind: ApisixRoute
 metadata:
-  name: httpbin-route
+ name: httpbin-route
```

Comment:
We should use two space indent

Reply:
Updated. I copied this from e2e test cases and did not check the indent. Updated all single spaces to double spaces.